src/openttd.cpp

```diff
@@ -395,19 +396,16 @@ struct AfterNewGRFScan : NewGRFScanCallback {
 	char *network_conn;                ///< Information about the server to connect to, or nullptr.
 	const char *join_server_password;  ///< The password to join the server with.
 	const char *join_company_password; ///< The password to join the company with.
-	bool *save_config_ptr;             ///< The pointer to the save config setting.
 	bool save_config;                  ///< The save config setting.
```

Comment:
Is it possible to also remove this?
Set _save_config just before "VideoDriver::GetInstance()->MainLoop();"?

Reply:
Hmm, probably not. There is a lot more magic going on here with joining servers.

Reply:
No, this is not possible, as the GUI is already running when the NewGRF scan is running. The idea of this delayed changing of `save_config` is that if for what-ever reason the `MainLoop` is aborted before the read configuration is loaded, a partial is not written to disk.